includes/Core/REST_API/REST_Routes.php

```diff
@@ -31,57 +35,17 @@
  * @since 1.0.0
  * @access private
  * @ignore
+ *
+ * @property-read Authentication $authentication Authentication instance.
+ * @property-read Context        $context        Plugin context.
+ * @property-read Modules        $modules        Modules instance.
+ * @property-read Transients     $transients     Transients API instance.
```

Comment:
This also confirms my comment above: Maintaining lists of those properties will be painful in the long run. The DI container itself will contain a lot more than those 4, and it will change. I'm fairly sure this will be a maintenance burden, and it won't help since we'll forget them and they'll be out of date quickly.

Reply:
@felixarntz this list of `@property-read` tags is not intended to list all values and services defined in the DI container. We use these annotations only for values and services that are needed in a particular class and only for the purpose of having autocomplete working in our IDEs.

Reply:
In other words, we don't need to update this list if we add a new service or a value.

Reply:
That's fair. Still, I'd prefer to do away with the magic getter and use a more explicit approach, like a method.